src/components/Proposals/ProposalDetails/DisputableActions.js

```diff
@@ -46,9 +46,9 @@ function DisputableActions({ status, submitter }) {
       <>
         <Button
           mode="strong"
-          disabled
-          wide
           label="Challenge proposal"
+          disabled={!connectedAccount}
```

Comment:
I think we should keep this button disabled (despite we have the real connectedAccount) until it can perform the action

Reply:
Sounds good!